packages/components/typography/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@buoysoftware/anchor-typography",
+  "version": "0.0.0",
```

Comment:
Are all the version starting here opposed to 0.0.1?

Reply:
When I run lerna publish it bumps the version then.